src/utils/googleMaps/markers/facility/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Facilities } from "../../../../models/facility";
+
+import { maps } from "../..";
+import { getIcon } from "./icons";
+
+export const markerRef = {
+  current: {} as Record<string, google.maps.Marker>,
+};
+
+export const setFacilities = (facilities: Facilities): void => {
+  if (!maps.mapRef.current) {
+    return;
+  }
+  const markers = Object.fromEntries(
+    Object.entries(facilities).map(([key, FacilityData]) => [
+      key,
+      new google.maps.Marker({
+        icon: getIcon(),
+        label: {
+          text: FacilityData.target.description,
+          fontFamily: "Roboto",
```

Comment:
question: I am not sure if every system supports `Roboto` font well.

Reply:
Change to fontFamily: "'Helvetica', 'Arial', 'sans-serif'", 
sans-serif is the default font-family in browser, without the first two items it would be the guarantee.